procurement_auto_create_group_carrier/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Procurement Auto Create Group Carrier",
+    "Summary": """Adds the delivery carrier in the procurement group data.""",
+    "version": "13.0.1.0.0",
+    "development_status": "Alpha",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-workflow",
+    "category": "Warehouse Managemen",
+    "depends": [
+        "stock_picking_group_by_partner_by_carrier",
+        "procurement_auto_create_group",
+    ],
+    "installable": True,
```

Comment:
auto_install?

Reply:
Should it be in `OCA/stock-logistics-warehouse` (alongside `procurement_auto_create_group`) or in `OCA/stock-logistics-workflow`?